src/components/ImportGithub.js

```diff
@@ -0,0 +1,60 @@
+import { Fragment, useState } from 'react'
+import { div, h, h2 } from 'react-hyperscript-helpers'
+import { ButtonPrimary } from 'src/components/common'
+import { TextInput } from 'src/components/input'
+import { submitMethod } from 'src/components/method-common'
+import { withBusyState } from 'src/libs/utils'
+
+// { tooltip: 'Link must start with https://github.com or https://raw.githubusercontent.com' },
```

Comment:
Is this meant to be commented out up here?

Reply:
Ah, thanks for catching. I meant to ask it as a question if we should add an info icon with a tooltip? I think it would be beneficial to users so they know what a valid link looks like. What do you think?